modules/compute-vm/README.md

```diff
@@ -31,6 +31,34 @@ module "simple-vm-example" {
 
 ```
 
+### Spot VM
+
+[Spot VMs](https://cloud.google.com/compute/docs/instances/spot) are ephemeral compute instances suitable for batch jobs and fault-tolerant workloads. Spot VMs provide new features that [preemptible instances](https://cloud.google.com/compute/docs/instances/preemptible) do not support, such as the absence of a maximum runtime.
+
+```hcl
+module "spot-vm-example" {
+  source     = "./modules/compute-vm"
+  project_id = var.project_id
+  zone     = "europe-west1-b"
+  name       = "test"
+  options = {
+    allow_stopping_for_update = true
+    deletion_protection       = false
+    preemptible               = true
```

Comment:
do we need both `preemptible` and `spot`? or should this be either/or?

Reply:
or even better, replacing both with a single option that takes `SPOT` and `PREEMPTIBLE` as args.